modules/platform/nuxeo-platform-document-routing/nuxeo-routing-api/src/main/java/org/nuxeo/ecm/platform/routing/api/DocumentRoutingService.java

```diff
@@ -622,4 +622,19 @@ void grantPermissionToTaskDelegatedActors(CoreSession session, String permission
      */
     boolean isWorkflowModel(final DocumentRoute documentRoute);
 
+    /**
+     * Returns true if the workflowModel can be started on the document list
```

Comment:
`Returns {@code true} if the workflow model can be started on the document list.`

Reply:
done

Reply:
Thanks. Just a 👍 on my comment and resolving the conversation is enough notification for me 😀